MVP/sample_job.yaml

```diff
@@ -0,0 +1,29 @@
+- job-template:
+    name: '{name}-{component}'
+    defaults: defaults-build
+    #node: 'jslave-of-your-own'
+    properties:
+        - ownership:
+            owner: lilu
+            co-owners:
+                - lilu
+                - lilu
```

Comment:
It's expected as notification "cc" recipients (e.g. dev counterpart or team lead if necessary).